### ISSUE TITLE
Menu bar and tool bar have dark text and icons on dark background

### ISSUE THREAD

**Issue by evgeniy-harchenko** [vinceliuice/Qogir-kde#56]
Some apps have unreadable menu bar and tool bar while using standard color scheme (Qogir).
![image](https://github.com/vinceliuice/Qogir-kde/assets/42656259/801ba1dd-071b-4d5d-ab6b-b53dc5c01c90)

Examples.

Krusader:
![krusader](https://github.com/vinceliuice/Qogir-kde/assets/42656259/43cd5506-3afe-4215-99ed-f0aca1170fe1)

Gittyup:
![gittyup](https://github.com/vinceliuice/Qogir-kde/assets/42656259/4bdee7ae-3e1f-45d5-9f51-618524524aa4)

Plasma (e.g. Download New Plasma Widgets):
![download_plasma_widgets](https://github.com/vinceliuice/Qogir-kde/assets/42656259/a1215d7b-1e08-49e0-9003-c9ca99721df8)

There is no problem while using light (QogirLight) or dark (QogirDark) color scheme.

**Comment by SeanyMCP** [comment]
Hello, has this issue been looked into ? It occurs in many regular apps in KDE such as dolphin, octopi, kontact suite...

**Comment by evgeniy-harchenko** [comment]
There are no poblems in Dolphin, tested both in Plasma 5 and 6. I can't test others now, but more likely they don't have this problem too. It seems you installed Kvantum not properly or you are using wrong theme here, so check it firstly.

**Comment by SeanyMCP** [comment]
So sorry, Kvantum wasn't installed. I didn't realise kvantum was needed for qogir. The github mentions kvantum "for better looking", thus I didn't see it as necessary.

**Comment by evgeniy-harchenko** [comment]
Yes, I guess it is better to change theme description. Anyway, all these problems can be solved by using color themes without `[Colors:Header]` section, you can find them in this PR: https://github.com/vinceliuice/Qogir-kde/pull/65
